apps/xprof_core/test/xprof_core_vm_info_tests.erl

```diff
@@ -69,6 +69,39 @@ get_available_funs_test_() ->
       end}
     ].
 
+get_called_funs_test_() ->
+    [{"Functions without calls return empty list",
+     fun() ->
+             ?assertEqual([], ?M:get_called_funs({?MODULE, get_called_funs_test_, 0}))
+     end},
+     {"Lists calls from simple functions in standard library",
+     fun() ->
+             ?assertEqual([{lists, reverse, 2}], ?M:get_called_funs({lists, reverse, 1})),
+             ?assertEqual([{erlang, nif_error, 1}], ?M:get_called_funs({lists, reverse, 2}))
```

Comment:
great testcases :) we need to filter `erlang:nif_error/1` out as it only indicates that `lists:reverse/2` is a BIF (implemented in C) so there is no further erlang functions called by it. 

This reminds me that we also need to filter out local functions based on `ModeCB:hidden_function(F)` (see `xprof_core_vm_info:get_all_functions/2`)